packages/components/src/MultiSelect/MultiSelect.spec.tsx

```diff
@@ -30,6 +30,7 @@ const MultiSelectWrapper = ({
     <MultiSelect
       items={defaultOptions}
       label="Jalapeno"
+      description="A short description"
```

Comment:
Oops missed this. Once again since it's optional (same logic for tests as stories), we don't want it there always 😬